sspl_test/alerts/os/test_systemd_service_actuator.py

```diff
@@ -21,12 +21,19 @@
 from messaging.ingress_processor_tests import IngressProcessorTests
 from messaging.egress_processor_tests import EgressProcessorTests
 from common import check_sspl_ll_is_running
+from framework.utils.conf_utils import Conf, GLOBAL_CONF, NODE_ID_KEY
+from framework.base.sspl_constants import DEFAULT_NODE_ID
 
 RESOURCE_TYPE = "node:sw:os:service"
 
 def init(args):
     pass
 
+def get_current_node_id():
+    """Get current node id."""
+    node_id = Conf.get(GLOBAL_CONF, NODE_ID_KEY, DEFAULT_NODE_ID)
+    return node_id
+
```

Comment:
Hi, why are we redefining this function?? you defined it already in sspl_test/common.py, right?

Reply:
Yes, removed it from all other file and importing it from common.py